airflow/configuration.py

```diff
@@ -421,6 +421,25 @@ def parameterized_config(template):
 conf = AirflowConfigParser()
 conf.read(AIRFLOW_CONFIG)
 
+if conf.getboolean('webserver', 'rbac'):
+    with open(os.path.join(_templates_dir, 'default_webserver_config.py')) as f:
+        DEFAULT_WEBSERVER_CONFIG = f.read()
+
+    WEBSERVER_CONFIG = AIRFLOW_HOME + '/webserver_config.py'
+
+    if not os.path.isfile(WEBSERVER_CONFIG):
+        log.info('Creating new FAB webserver config file in: %s', WEBSERVER_CONFIG)
+        with open(WEBSERVER_CONFIG, 'w') as f:
+            f.write(DEFAULT_WEBSERVER_CONFIG)
+
+    if 'WEBSERVER_CONFIG' not in os.environ:
+        if os.path.isfile(expand_env_var('~/webserver_config.py')):
```

Comment:
This seems like an odd file location to open -- having a webserver_config.py in a home directory is somewhat confusing. GIven the ways of overloading  the file to read can we remove this and not look for this file automatically?